web/packages/teleport/src/Console/DocumentSsh/useSshSession.ts

```diff
@@ -66,9 +106,23 @@ export default function useSshSession(doc: DocumentSsh) {
             const data = JSON.parse(payload);
             data.session.kind = 'ssh';
             data.session.resourceName = data.session.server_hostname;
+            setSession(prev => ({
+              ...prev,
+              moderated: data.session.moderated,
+            }));
```

Comment:
We weren't setting any returned session data at all here previously so no server fetched data was being stored. I couldn't find a reason for this, but just in case, i _only_ added the moderated flag. However, i'd like to just set all returned data here. In the future, we can have a lot more things held on the session, such as already active fileTransferRequests and other things. Thoughts?